testcharms/charm-hub/charms/juju-qa-test/README.md

```diff
@@ -6,46 +6,46 @@ A container-based V2 metadata charm to use in testing juju.
 
 ## Usage
 
-Basic deploy: <br>
```

Comment:
Can you also rename 'charm-hub' to 'charmhub' as part of this? It isn't a huge deal, but it is the official Canonical branding (one word, no hyphens or caps).